gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImplCreateTest.java

```diff
@@ -215,82 +205,6 @@ public void close() {
     assertThat(thrown).hasCauseThat().isEqualTo(fakeException);
   }
 
-  /**
-   * Test handling when the parent thread waiting for the write operation to finish via the close
-   * call is interrupted, that the actual write is cancelled and interrupted as well.
-   */
-  @Test
-  public void testCreateObjectApiInterruptedException() throws Exception {
```

Comment:
these test cases are used against apairy writer as well as java storage writer. We should be careful while removing test cases from here.

Reply:
Good point. Missed that.